set.mm

```diff
@@ -37384,11 +37403,21 @@ technically classes despite morally (and provably) being sets, like ` 1 `
   df-pss $a |- ( A C. B <-> ( A C_ B /\ A =/= B ) ) $.
 
   ${
-    $d x A $.  $d x B $.
+    $d x y A $.  $d x y B $.
     $( Alternate definition of the subclass relationship between two classes.
        Definition 5.9 of [TakeutiZaring] p. 17.  (Contributed by NM,
-       8-Jan-2002.) $)
+       8-Jan-2002.)  Avoid ~ ax-10 to ~ ax-12 .  (Revised by SN,
```

Comment:
```suggestion
       8-Jan-2002.)  Avoid ~ ax-10 , ~ ax-11 , ~ ax-12 .  (Revised by SN,
```
Easier to text-search, or in case axioms are renamed, reordered, etc.